### ISSUE TITLE
PHP 8 compatibility

### ISSUE THREAD

**Issue by nstCactus** [jamesedmonston/graphql-authentication#65]
Currently the plugin is not compatible with PHP 8 due to a dependency to [`abraham/twitteroauth:^1.0`](https://packagist.org/packages/abraham/twitteroauth#1.2.0).

The [latest version of `abraham/twitteroauth`](https://packagist.org/packages/abraham/twitteroauth#3.1.0) seems compatible with PHP 8.0. I have no idea of the amount of work required to make the plugin compatible with the latest version, but it would help in PHP 8 compatibility.

Composer returns the following error when trying to install on PHP 8: 
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - jamesedmonston/graphql-authentication[1.10.0, ..., 1.10.3] require abraham/twitteroauth ^1.0 -> satisfiable by abraham/twitteroauth[1.0.0, 1.0.1, 1.1.0, 1.2.0].
    - abraham/twitteroauth[1.0.0, ..., 1.0.1] require php ^5.6 || ^7.0 || ^7.1 || ^7.2 -> your php version (8.0.7) does not satisfy that requirement.
    - abraham/twitteroauth 1.1.0 requires php ^7.2 || ^7.3 -> your php version (8.0.7) does not satisfy that requirement.
    - abraham/twitteroauth 1.2.0 requires php ^7.2 || ^7.3 || ^7.4 -> your php version (8.0.7) does not satisfy that requirement.
    - Root composer.json requires jamesedmonston/graphql-authentication ^1.10 -> satisfiable by jamesedmonston/graphql-authentication[1.10.0, 1.10.1, 1.10.2, 1.10.3].
```

**Comment by jamesedmonston** [comment]
@nstCactus thanks for the heads up! I'll get an env set up locally and check it out.

**Comment by holmey** [comment]
PHP 8 support would be great. I configured an alias for [abraham/twitteroauth](https://packagist.org/packages/abraham/twitteroauth#3.1.0) so this plugin is used in the latest version.
But when this issues was resolved i figured out that also [facebook/graph-sdk](https://github.com/facebookarchive/php-graph-sdk) does not have support for PHP 8.

**Comment by jamesedmonston** [comment]
I've just released `1.11.0` includes this fix.
